### ISSUE TITLE
declaring a design with nonexistent object -> hard to parse error

### ISSUE THREAD

**Issue by graemeblair** [DeclareDesign/DeclareDesign#65]
```
> declare_design(oij)
Error: `lang` must quote a lang
```

**Comment by nfultz** [comment]
Needs better logic around autoboxing calls - this is also related to using untagged functions as DGP alluded too in #10 and #114.

**Comment by jaspercooper** [comment]
Error has changed to 

```
> declare_design(oij)
Error: `call` must quote a call 
```

**Comment by graemeblair** [comment]
Think we need to fix this before pushing to CRAN

**Comment by nfultz** [comment]
Fixed in #150,  error message is now 

```
> declare_design(oij)
 Error in value[[3L]](cond) : 
  Could not evaluate step 1 as either a step or call. 
```
